### PR TITLE
Readds various admin verbs and timer noise

### LINKED ISSUE #195: timers dont make any sound
how can i fake terrorism now

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: Ported various admin verbs, including fill-hull-breach, clean-air, and also added humanrunes (search for humanrunes in the spawn menu.)
add: There is now a sound when a timer device is counting down.
/:cl:

Fixes #195

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1174,3 +1174,45 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
...
+
+/client/proc/fill_breach()
+	set name = "Fill Hull Breach"
+	set category = "Special Verbs"
+	set desc = "Spawns plating over space breachs"
+	var/size = input("How big?", "Input") in list(5, 10, "Cancel")
+	if(size == "Cancel")
+		return 0
+	for(var/turf/open/space/T in range(size))
+		T.ChangeTurf(/turf/open/floor/plating)
```
Comment (ghost) [inline code/modules/admin/verbs/randomverbs.dm]: add a `CHECK_TICK` here

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -1174,3 +1174,45 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
...
+	set desc = "Cleans the air in a radius of harmful gasses like plasma and n2o "
+	var/size = input("How big?", "Input") in list(5, 10, 20, "Cancel")
+	if(size == "Cancel")
+		return 0
+	for(var/turf/open/T in range(size))
+		if(T.air)
+			var/datum/gas_mixture/A = T.air
+			T.overlays.Cut()
+			if(A)
+				A = initial(A)
```
Comment (ghost) [inline code/modules/admin/verbs/randomverbs.dm]: add a `CHECK_TICK` here

File: code/modules/research/xenobiology/xenobiology.dm [hunk 3]
```diff
@@ -539,6 +539,52 @@
 	log_admin("[key_name(G)] was made a golem by [key_name(user)].")
 	qdel(src)
 
+//Ghost-to-Human rune for badmins
+
+/obj/effect/golemrune/humanrune		//Summons humans instead of golems - for admins exclusively
+	name = "Admin Rune"
+	desc = "A rune used by badmins to turn ghosts into humans. Oh boy."
+
+	New()
```
Comment (ghost) [inline code/modules/research/xenobiology/xenobiology.dm]: Please use full paths. `/obj/effect/golemrune/humanrune/New()`

File: code/modules/research/xenobiology/xenobiology.dm [hunk 4]
```diff
@@ -539,6 +539,52 @@
...
+		..()
+		SSobj.processing |= src
+
+/obj/effect/golemrune/humanrune/process()
```
Comment (ghost) [inline code/modules/research/xenobiology/xenobiology.dm]: Do we really need our own process? Can't we piggy back off the `golemrune` process?

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by KittyKathy** [comment]
Should make it a option for the timer to make sound as you can't really use it for bombing when everyone will run away from ticking sounds. Like drop a grenade in a container on the shuttle with a timer everyone hears ticking finds it then spaces it or just runs away, not really useful.

File: code/modules/research/xenobiology/xenobiology.dm (hunk 4 above)
Comment (tortellinitony) [reply code/modules/research/xenobiology/xenobiology.dm]: I'll check

**Comment by tortellinitony** [comment]
it's not the same sound as the bomb now

**Comment by JohnMcLaime** [comment]
Doesnt matter if it is not the same sound as a bomb, people will recognise it and run away, timer is already outclassed by signaller, why make it even more shit? Just for the sake of boring pranks?

**Comment by tortellinitony** [comment]
yes

**Review by j-mie — APPROVED** [review]

**Pull request merged**